targets/qtpy-rp2040.json

```diff
@@ -0,0 +1,11 @@
+{
+    "inherits": [
+        "rp2040"
+    ],
+    "serial-port": ["acm:239a:80f1"],
```

Comment:
```suggestion
    "serial-port": ["acm:239a:80f7"],
```

Reply:
Updated serial port to match updated usb_PID of 0x80F7